src/components/HomePage.jsx

```diff
@@ -36,6 +36,4 @@ export default function HomePage({ loginUser }) {
       </div>
     </div>
   );
-}
-
-export default HomePage;
\ No newline at end of file
+}
```

Comment:
I've seen this on a lot of your PRs - it'd be nice to configure your editor to leave a newline at the end (this is pretty standard for JS).

Reply:
Done. In VSCode you add `"files.insertFinalNewline": true,` to your user settings cc @andrewgordstewart .